docs/github.md

```diff
@@ -0,0 +1,90 @@
+# Using Jujutsu with GitHub and GitLab Projects
+
+This guide assumes a basic understanding of either Git or Mercurial. 
+
+## Set up an SSH key
+As of December 2022 it's recommended to set up an SSH key to work with Github
+projects. See [Github's Tutorial][gh]. This restriction may be lifted in the 
+future, see [issue #469][http-auth] for more information and progress on authenticated http. 
+
+
+## Basic workflow
+The simplest way to start with Jujutsu, is creating a stack of commits, before 
+creating any branch. 
+
+```shell script
+# Start a new commit off of `main`
+$ jj new main
+# Refactor some files, then add a description and start a new commit
+$ jj commit -m 'refactor(foo): restructure foo()'
+# Add a feature, then add a description and start a new commit
+$ jj commit -m 'feat(bar): add support for bar'
+# Create a branch so we can push it to GitHub
+$ jj branch create bar -r @-
+# Push the branch to GitHub (pushes only `bar`)
+$ jj git push
+```
+
+While it's possible to create a branch and commit on top of it in a Git like 
+manner, it's not recommended, as no further commits will be placed on the branch. 
+
+## Updating the repository.
+As of December 2022, Jujutsu has no equivalent to a `git pull` command.
+Until such a command is added, you need to use `jj git fetch` followed by a 
+`jj git rebase -d $main_branch` to update your changes. 
+
+## Working in a Git co-located repository
+After doing `jj init --git-repo=.`, git will be in a detached HEAD state,
+which is unusual, as git mainly works with branches. 
+In a co-located repository, `jj` isn't the source of truth. But Jujutsu
+allows a incremental migration, as `jj commit` updates the HEAD of the git
+repository. 
+
+
+Although `jj git import` and `jj git export` exist for co-located repositories,
+they should only be used in case of an unrecoverable error.
+
+```shell script
+$ nvim docs/tutorial.md
+$ # Do some more work.
+$ jj commit -m "Update tutorial"
+$ jj branch create doc-update
+$ # Move the previous revision to doc-update.
+$ jj branch set doc-update @- 
+$ jj git push 
+```
+
+## Working in a Jujutsu repository
+In a Jujutsu repository, the workflow is simplified. If there's no need for 
+explicitly named branches, you just can generate one for a change. As Jujutsu 
+is able to create a branch for a revision.
+
+```shell script
+$ ij . 
```

Comment:
what is `ij`?

Reply:
IntelliJ. It's a similar feature to vscode's `code .`, which opens the current directory in vscode.

Reply:
Please put a comment in the example or use a different command that is well known.

Reply:
Done, it's now  `code .`

Reply:
Does `code` start VS Code? I think it's better to not specify a particular code editor here and just have a comment like in the example above.